Octokit/Models/Request/NewCommitCommentReaction.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class NewCommitCommentReaction
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="NewCommitCommentReaction"/> class.
+        /// </summary>
+        /// <param name="content">The reaction type.</param>
+        public NewCommitCommentReaction(Reaction content)
+        {
+            switch (content){
```

Comment:
I didn't pull down the code to test it, but shouldn't you be able to just do `Content = content`?